Shared/View Controllers/RelistenTableViewController.swift

```diff
@@ -369,16 +369,18 @@ public class RelistenAsyncTableController<TData> : RelistenBaseAsyncTableontroll
             load()
         }
         
-        if useCache, let res = resource {
```

Comment:
There's a good chance this is the wrong change. I don't think I understand the whole data loading flow here well enough, so let me know if I missed something subtle here.